dev/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "require": {
+        "vierbergenlars/php-semver": "^3.0",
+        "symfony/console": " ^3.0",
+        "symfony/yaml": " ^3.3||^6.0",
+        "symfony/process": "^6.0",
+        "google/cloud": "^0.100.0",
+        "symfony/filesystem": "^6.0",
+        "twig/twig": "^3.5"
+    },
+    "autoload": {
+        "psr-4": {
+            "Google\\Cloud\\Dev\\": "src"
+        }
+    },
+    "require-dev": {
+        "phpspec/prophecy": "^1.17",
+        "phpunit/phpunit": "4.8|^5.0|^8.0",
+        "swaggest/json-schema": "^0.12.0",
+        "yoast/phpunit-polyfills": "^1.0"
+    },
+    "repositories": {
+        "google-cloud": {
+            "type": "path",
+            "url": "../",
+            "options": {
+                "versions": {
+                    "google/cloud": "0.100.0"
```

Comment:
This is ~2 years old, [ref](https://github.com/googleapis/google-cloud-php/releases/tag/v0.100.0). Better to have a more recent version?

Reply:
Looks like I can also name it something else. So how about `dev-parent`? That makes it even more obvious that it's not pulling from an actual tagged version...